README.md

```diff
@@ -571,28 +619,32 @@ The CirrusMDSDK can you provide you with a view controller that displays various
 
 THIS VIEW CONTROLLER SHOULD NEVER BE PRESENTED TO AN END USER. It is for debugging purposes only.
 
+The debug view controller can be accessed and presented manually or it can be accessed in Settings if `showDebugInSettings` is set to true on your `CirrusMDSDKConfig`
```

Comment:
Also net new documentation here around the new showDebugInSettings config option.